src/txn/txn.c

```diff
@@ -1155,7 +1155,7 @@ __txn_commit_timestamps_assert(WT_SESSION_IMPL *session)
             continue;
         }
 
-        /* Search for prepared updates, so that they will be restored, if moved to lookaside. */
+        /* Search for prepared updates. */
```

Comment:
I think it is better to keep the complete sentence and just replace `lookaside` by `history store`.

Reply:
No. then that is wrong. We are not searching the history store here. Before with lookaside, we need to reconstruct the update chain from data in the lookaside but now we only search data store.